portsyncd/linksync.cpp

```diff
@@ -200,17 +197,12 @@ void LinkSync::onMsg(int nlmsg_type, struct nl_object *obj)
             return;
         }
 
+        /* Host interface is created */
         if (!g_init && g_portSet.find(key) != g_portSet.end())
         {
-            /* Bring up the front panel port as the first place*/
-            /* TODO: handle system retur code, if-block is just a warning suppressor */
-            if (system(("/sbin/ifup --force " + key).c_str()))
-                ;
             g_portSet.erase(key);
```

Comment:
>g_portSet [](start = 12, length = 9)

the description in line 190 is not complete. Check if the port is in the PORT_TABLE.

if the port is in the PORT_TABLE, then what? If not, then what?

honestly saying, i do not quite understand the block 201, if a new link is detected and is in g_portSet, then you remove the port from g_portSet. What is g_portSet?